CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+Version 0.8.0 (UNRELEASED)
+--------------------------
+
+- Changes ``delete`` to prevent workflow hard deletion.
```

Comment:
Please put all "Adds" before "Changes" so that they are grouped together.  The usual order is new/fixed/improved... or perhaps we could better switch it to new/improved/fixed, so Adds/Changes/Fixes.